spec/support/shared_contexts.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.shared_context 'with logged in user' do
```

Comment:
**[Minor]** It looks like the indentation on this file is different from the others, i.e. 4 spaces vs. 2. You can probably add a linter/formatter like [rubocop](https://github.com/rubocop/rubocop), [standard](https://github.com/standardrb/standard) or maybe even [prettier with a ruby plugin](https://github.com/prettier/plugin-ruby) to get a consistent style applied to all of your files. 😄